tools/bump_version.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import argparse
+import subprocess
+from collections.abc import Sequence
+
+import tomllib
+
+
+def extract_packages(package_specs: list[str]) -> set[str]:
+    names = set()
+    for spec in package_specs:
+        parts = spec.split(">=")
+        if len(parts) == 1:
+            raise SystemExit(
+                f"Only >= is allowed for packages in pyproject.toml. Offfending spec: {spec}"
```

Comment:
Actually umm, if we want to enforce `>=` then I can move this part to lint_requirements in part 3: https://github.com/getsentry/sentry/pull/96817/files#diff-997a1f89775776afed67542c4a9ae2ed0b1a687de8f33320f94da9e6a71dd171

Reply:
i'll just leave this here for now, won't hurt